yarn-project/noir-contracts/src/contracts/card_game_contract/Nargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "card_game_contract"
+authors = [""]
+compiler_version = "0.1"
+type = "contract"
+
+[dependencies]
+aztec = { path = "../../../../noir-libs/noir-aztec" }
+value_note = { path = "../../../../noir-libs/value-note"}
+easy_private_state = { path = "../../../../noir-libs/easy-private-state"}
```

Comment:
question - is easy_private_state just a dep for `aztec`?  i dont think its explicitly `use`d in the contract